src/discordtipbot/discordbot.cr

```diff
@@ -533,6 +535,44 @@ class DiscordBot
     end
   end
 
+  def glucky(msg : Discord::Message, cmd_string : String)
+    return reply(msg, "**ERROR**: This command doesn't work in DMs") if private?(msg)
+
+    cmd_usage = "#{@config.prefix}lucky [amount]"
+
+    # cmd[0]: command, cmd[1]: amount, cmd[2]: people"
+    cmd = cmd_string.split(" ")
+
+    return reply(msg, cmd_usage) unless cmd.size > 2
+
+    amount = amount(msg, cmd[1])
+    return reply(msg, "**ERROR**: You have to specify an amount! #{cmd_usage}") unless amount
+
+    people = cmd[2].to_i? # Number of people to give money to.
```

Comment:
Same goes for the comment here. Please put them above the line they describe (90% of the time that is)